src/screens/Subscriptions/AfterUnsubscribe.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { createRef } from 'react';
+import { Wrapper, Header, BodyCopy } from './Subscriptions.style';
+import { EventAction, EventCategory } from 'components/Analytics';
+import ExternalLink from 'components/ExternalLink';
+import { LargeFilledButton } from 'components/ButtonSystem';
+
+const AfterUnsubscribe = () => {
+  const params = new URLSearchParams(window.location.search);
+  const email = params.get('email') || '';
+  const formRef = createRef<HTMLFormElement>();
+
+  const formAction = getFormActionLink(email);
+
+  const onSubscribeClick = async () => {
+    if (formRef?.current) {
+      formRef.current.action = await formAction;
+      formRef.current.submit();
+    }
+  };
+
+  return (
+    <Wrapper>
+      <Header>One last thing...</Header>
+      <BodyCopy>
+        <p>
+          Hi there! It looks like you're leaving us. We're sorry to see you go
+          and wanted to thank you for being a subscriber and supporter of Covid
+          Act Now.
+        </p>
+        <p>
+          Our organization will be working on some exciting new projects in the
+          future. Our mission is the same: to help people make informed
+          decisions by focusing on{' '}
+          <ExternalLink href="https://covidactnow.org/about">
+            data, transparency and accessibility
+          </ExternalLink>
+          .
+        </p>
+        <p>
+          If you would like to be alerted about our new ventures, click the
+          button below. And if there are any issues, topics or current events
+          that you think could benefit from the Covid Act Now approach please
+          let us know at{' '}
+          <a href="mailto:info@covidactnow.org">info@covidactnow.org</a>.
+        </p>
+      </BodyCopy>
+      <LargeFilledButton
+        onClick={onSubscribeClick}
+        trackingCategory={EventCategory.ENGAGEMENT}
+        trackingAction={EventAction.SUBSCRIBE}
+        trackingLabel="Future Projects (after-unsubscribe)"
+      >
+        Alert me about future projects
+      </LargeFilledButton>
+
+      <form
+        ref={formRef}
+        className="js-cm-form"
+        id="subForm"
+        method="POST"
+        data-id={CREATESEND_DATA_ID}
+      >
+        <input
+          className="js-cm-email-input qa-input-email"
+          id="fieldEmail"
+          name="cm-jrjjlhr-jrjjlhr"
+          style={{ display: 'none' }}
```

Comment:
instead of display: none - this should just be `type="hidden"`

Reply:
Oh, good call!